openreview/venue/venue.py

```diff
@@ -200,8 +200,15 @@ def get_homepage_options(self):
         options['contact'] = self.contact
         return options
 
+    def get_submission_venue_id(self, submission_invitation_id=None):
+        if submission_invitation_id:
+            return f'{self.venue_id}/{submission_invitation_id}'
```

Comment:
Why would we pass `submission_invitation_id` here? What would this venue id look like?

Reply:
I did it in case that in the future we may have multiple submission invitations.

Reply:
oh I see! I just got confused because it seems like the `submission_invitation_id` would already include the venue_id

Reply:
ohhh you are right! the parameter should be called "submission_name"! let me fix it.